frontera/contrib/messagebus/zeromq/socket_config.py

```diff
@@ -1,29 +1,33 @@
 # -*- coding: utf-8 -*-
+import socket
 
 
 class SocketConfig(object):
-
-    hostname = None
-    base_port = None
+    __ip_addr = None
```

Comment:
@lljrsr this code isn't mine. I'm just looking after it, because I can't just leave it for free modification to everyone.
I think these private methods can be used in possible descendants.